resources/en/training/t-6.training.bxb

```diff
@@ -0,0 +1,10 @@
+train (t-6dxbiiqin5l3ms5abnxuko1dq) {
```

Comment:
I don't think we should have training in the dice capsule since the QSG teaches people how to add it.